.editorconfig

```diff
@@ -1,15 +1,17 @@
 # top-most EditorConfig file
 root = true
 
+[*]
```

Comment:
I made the changes to .editorconfig already, if you could remove these changes and rebase so this file isn't included that'd be great.